daisy_workflows/build-publish/windows_media/windows-install-media.wf.json

```diff
@@ -0,0 +1,100 @@
+{
+  "Name": "windows-install-media",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "Vars": {
+    "workflow_root": {
+      "Value": "/workflows",
+      "Description": "Root of github workflows, defaults to /workflows in the container."
+    },
+    "gcs_url": {
+      "Required": true,
+      "Description": "The GCS path that image raw file exported to."
+    },
+    "gcs_resource_path": "gs://gce-image-build-resources/windows",
+    "media_disk": "media-disk",
+    "image_prefix": "windows-install-media",
+    "iso_name_2022": "SW_DVD9_Win_Server_STD_CORE_2022__64Bit_English_DC_STD_MLF_X22-74290.ISO",
+    "iso_name_2019": "SW_DVD9_Win_Server_STD_CORE_2019_1809.1_64Bit_English_DC_STD_MLF_X22-02970.ISO",
+    "iso_name_2016": "SW_DVD9_Win_Server_STD_CORE_2016_64Bit_English_-4_DC_STD_MLF_X21-70526.ISO",
+    "iso_name_2012r2": "SW_DVD9_Windows_Svr_Std_and_DataCtr_2012_R2_64Bit_English_-4_MLF_X19-82891.ISO"
+  },
+  "Sources": {
+    "prepare_install_media.ps1": "./prepare_install_media.ps1",
+    "components": "./components/",
+    "iso/Win_Server_STD_CORE_2022_64Bit_English.iso": "${gcs_resource_path}/iso/${iso_name_2022}",
```

Comment:
These cannot be referenced externally. The values need to be pulled from secret manager and passed into this workflow by concourse.